packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/About.5.doc.mdx

```diff
@@ -66,8 +67,6 @@ The order in which the rows are provided will be the order in which they are ren
 |---|---|---|---|---|
 |**id**|string|required|none|An identifier for the row. This identifier should be unique across the set of rows provided to the Grid.|
 |**ariaLabel**|string|optional|none|A string identifier used to describe the row contents. This value will be used for accessibilty when announcing the row that is selected or deselected.|
-|**isSelectable**|bool|optional|none|A boolean indicating whether or not the row is selectable.|
-|**isSelected**|bool|optional|none|A boolean indicating whether or not the row should render as selected.|
```

Comment:
We still need the "isSelected" prop.

Reply:
Right, I need to add this back. I also realized I can use this in the example instead of individually selecting the cells.